sink/kafka.go

```diff
@@ -108,8 +110,9 @@ func (s *KafkaSink) Stop() {
 }
 
 // Put ..
-func (s *KafkaSink) Put(data []byte) error {
+func (s *KafkaSink) Put(key string, data []byte) error {
 	s.putCh <- data
+	s.key = key
```

Comment:
I'm not sure this is correct; you're sending the data through the channel `putCh` but mutating the sink to set the key, which I think will be incorrect. I think `putCh` will need to be modified to take both (as a struct, maybe) `data []byte` and also `key []string`.

Reply:
hm, you might be right. but it is working currently, i'll see what @jippi or @josegonzalez say.

Reply:
@wyattanderson updated